src/AlgebraicGeometry/Schemes/AffineSchemeOpenSubscheme/Rings/Methods.jl

```diff
@@ -6,6 +6,12 @@ function ==(R::AffineSchemeOpenSubschemeRing, S::AffineSchemeOpenSubschemeRing)
   return scheme(R)==scheme(S) && domain(R)==domain(S)
 end
 
+function Base.hash(R::AffineSchemeOpenSubschemeRing, h::UInt)
+  h = hash(scheme(R), h)
+  h = hash(domain(R), h)
```

Comment:
```suggestion
```
Frankly, I do not remember from the top of my head what `domain` of `R` is in the first place.

Reply:
I looked it up. Yes, it's fine.